packages/content-management/table_list/src/table_list_view.tsx

```diff
@@ -441,23 +462,30 @@ function TableListViewComp<T extends UserContentCommonSchema>({
     return null;
   }
 
+  const PageTemplate = withoutPageTemplateWrapper
+    ? (React.Fragment as unknown as typeof KibanaPageTemplate)
+    : KibanaPageTemplate;
+
   if (!fetchError && hasNoItems) {
     return (
-      <KibanaPageTemplate panelled isEmptyState={true} data-test-subj={pageDataTestSubject}>
+      <PageTemplate isEmptyState={true} data-test-subj={pageDataTestSubject}>
```

Comment:
You removed the `panelled` prop, wasn't it necessary?

Reply:
Yes, good catch, that was a mistake!